tr/README.md

```diff
@@ -4,40 +4,42 @@
 
 [![Sürümler](https://img.shields.io/badge/release-master-blue.svg)](https://github.com/PX4/PX4-Autopilot/releases) [![Tartışma](https://img.shields.io/badge/discuss-px4-ff69b4.svg)](https://discuss.px4.io//) [![Slack](../assets/site/slack.svg)](https://join.slack.com/t/px4/shared_invite/zt-si4xo5qs-R4baYFmMjlrT4rQK5yUnaA)

```

Comment:
@avcuenes Thank you, but we cannot take translations done in github. Can you revert all this.
We'd love your translations if there are enough people working on them to make it worth the cost of building a dedicated version - but they would need to be done in crowdin.  http://docs.px4.io/main/en/contribute/translation.html

Reply:
Thanks for your reply. I